portfolio/src/main/webapp/script.js

```diff
@@ -17,7 +17,7 @@
  */
 function addRandomGreeting() {
```

Comment:
Small nit, you may want to update the name of this method now that it includes TV shows and not greetings.